Section1_lab4_ShanHe_JoannaHuang_TiffanyJaya.Rmd

```diff
@@ -93,180 +94,152 @@ summary(data$wser)
 
 # Model 1: only the explanatory variables of key interest
 
-Based on the exploratory data analysis and goal of testing the "broken windows theory", our initial proposed model will include all variables related to stricter law enforcement policies: *prbarr*, *prbconv*, *prbpris*, *avgsen*, and *polpc*. Assuming the "broken windows theory" is valid, we expect generally negative coefficients for all variables.
+Based on the research question, our initial proposed model will include $crmrte$ as the dependent variable and all variables related to stricter law enforcement policies: $prbarr$, $prbconv$, $prbpris$, $avgsen$, and $polpc$ as independent variables. Assuming the "broken windows theory" is valid, we expect generally negative coefficients for all variables.
 
-To make *prbarr*, *prbconv* and *prbpris* interpretation more intuitive to understand, we decided to transform them into percentages, and will use these new variables moving forward.
-```{r}
-data$arrperc = data$prbarr * 100
-data$convperc = data$prbconv * 100
-data$prisperc = data$prbpris * 100
+Given that the histogram of $crmrte$ has a significant positive skew, we noted a log transformation may be suitable since its values are non-zero and positive. The same can be said about the independent variable $polpc$ where its histogram is positively skewed and its values are non-zero and positive.
+
+```{r out.width="49%", fig.align="center", fig.show="hold"}
+# before and after log transformation
+hist(data$crmrte); hist(log(data$crmrte))
+hist(data$polpc); hist(log(data$polpc))
 ```
 
-In our EDA, we noted that crmrte, prbarr, prbconv, avgsen, and polpc are positively skewed, thus we should consider taking the log. Since the values cannot be negative and have meaningful zero-points, this decision is valid. It is important to remember moving forward that the interpretation of arrperc, and convperc will be in terms of percentage point changes rather than percentage changes.
+Though $prbarr$, $prbconv$, and $prbpris$ are positively skewed as well, we decided against taking the log of these variables because log transformations can make values between 0 and 1 more extreme. We also kept $avgsen$ as is for easier interpretation. Plus, applying the log on $avgsen$ does not seem to affect the normality of the residual distribution.
```

Comment:
do we have proof for "Plus, applying the log on $avgsen$ does not seem to affect the normality of the residual distribution"? Also, if we do include qq plot to prove, instead of "the normality of the residual distribution", I think we should be looking at the "the normality of the variable". We haven't mentioned the model yet